src/lib/components/workflow/filter-search/datetime-filter.svelte

```diff
@@ -239,18 +152,117 @@
               bind:hour={startHour}
               bind:minute={startMinute}
               bind:second={startSecond}
-              bind:half={startHalf}
+              twelveHourClock={false}
             />
-            <div class="flex justify-end">
-              <Button variant="ghost" on:click={onApply}
-                >{translate('apply')}</Button
+          </div>
+        </MenuItem>
+        <MenuDivider />
+        <MenuItem>
+          <div class="flex flex-col gap-2">
+            <DatePicker
+              label={translate('end')}
+              on:datechange={onEndDateChange}
+              selected={endDate}
+              todayLabel={translate('today')}
+              closeLabel={translate('close')}
+              clearLabel={translate('clear-input-button-label')}
+            />
+            <TimePicker
+              bind:hour={endHour}
+              bind:minute={endMinute}
+              bind:second={endSecond}
+              twelveHourClock={false}
+            />
+          </div>
+        </MenuItem>
+      {:else}
+        <MenuItem on:click={() => (type = 'relative')}>
+          <div class="flex flex-col">
+            <label
+              class="flex flex-row items-center gap-2 cursor-pointer"
+              for="relative-time"
+            >
+              <input
+                on:click|stopPropagation={() => {
+                  type = 'relative';
+                }}
+                class="w-4 h-4 accent-gray-900"
+                type="radio"
+                checked={type === 'relative'}
+                id="relative-time"
+                tabindex="-1"
+              />
+              {translate('relative')}
+            </label>
+            <div class="ml-6 pt-2 flex gap-0">
+              <Input
+                label={translate('relative')}
+                labelHidden
+                id="relative-datetime-input"
+                bind:value={relativeTime}
+                placeholder="00"
+                error={error(relativeTime)}
+                unroundRight
+                class="h-10"
+              />
+              <Select
+                unroundLeft
+                bind:value={timeUnit}
+                id="relative-datetime-unit-input"
+                label={translate('time-unit')}
+                labelHidden
               >
+                {#each TIME_UNIT_OPTIONS as unit}
+                  <Option value={unit}>{unit} {translate('ago')}</Option>
+                {/each}
+              </Select>
+            </div>
+          </div>
+        </MenuItem>
+        <MenuDivider />
+        <MenuItem on:click={() => (type = 'absolute')}>
```

Comment:
We can't make `MenuItem`s full width because of the space reserved for the checkmark

<img width="440" alt="Screenshot 2023-09-01 at 4 03 53 PM" src="https://github.com/temporalio/ui/assets/15069288/e09a2c95-eb6f-476c-90e8-e231a4d3c210">


Do we want to update the conditional [here](https://github.com/temporalio/ui/blob/main/src/lib/holocene/menu/menu-item.svelte#L143-L147) to include the container `<div/>`? If we do, selecting something might alter the height of the option selected (e.g. if the text wraps).

Reply:
Yeah hmm. Maybe we need a different variant of MenuItem that isn't intended to be selected.

Reply:
[Added a check](https://github.com/temporalio/ui/pull/1613/commits/9d686f48e41bd5b1a40d2b2a543a0e63b7a54206) for if `selected` is being passed as a prop or not ✅